uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/SingleFuseProjectTest.java

```diff
@@ -278,11 +285,11 @@ public void testArchetype() {
 			assertTrue("Project '" + project + "' has something with Camel Editor", isEditorOK());
 			if (project.getTemplate()[project.getTemplate().length - 1].toLowerCase().contains("empty")
 					|| project.getTemplate()[project.getTemplate().length - 1].toLowerCase()
-							.contains("spring bean - spring dsl") && project.getRuntimeType().equals(EAP)) {
+							.contains("springa bean - spring dsl") && project.getRuntimeType().equals(EAP)) {
```

Comment:
why is it `springa`? a typo in tested code? Or a typo in this code not detected because the new jira annotation is deactivating the play of the test?

Reply:
Yes, it's a typo! Thank you for noticing.